pom.xml

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <parent>
+    <groupId>org.commonjava</groupId>
+    <artifactId>service-parent</artifactId>
+    <version>2</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.commonjava.indy.service</groupId>
+  <artifactId>indy-tracking-service</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <name>Indy :: Service :: Tracking</name>
+  <url>https://github.com/Commonjava/indy-tracking-service</url>
+  <description>Indy Tracking Service</description>
+  <inceptionYear>2022-2023</inceptionYear>
+
+  <scm>
+    <connection>scm:git:https://github.com/commonjava/indy-tracking-service</connection>
+    <developerConnection>scm:git:https://github.com/commonjava/indy-tracking-service</developerConnection>
+    <url>http://github.com/Commonjava/indy-tracking-service</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <properties>
+    <apiVersion>1</apiVersion>
+    <projectEmail>https://github.com/Commonjava/indy-tracking-service</projectEmail>
+
+    <atlas.version>1.1.1</atlas.version>
+    <eventmodel.version>1.0</eventmodel.version>
+    <indysecurity.verison>1.0</indysecurity.verison>
+
+    <cassandra.version>3.11.3</cassandra.version>
+    <infinispan.version>11.0.8.Final</infinispan.version>
```

Comment:
Not sure if tracking service is still depends on infinispan. Can you help have a check and remove it if no deps there?

Reply:
it is required later for listening and reacting to events. Removed for now.